### DISCUSSION TITLE
Using temperature and humidity sensor

### DISCUSSION THREAD

**Discussion by triatebr** [BIPES/BIPES#184] [General]
I created with blocks a connection to the DHT22 sensor using the ESP8266 board.
When I execute PLAY, I notice that the collected data does not change, even when I change the place.

blocks: [link](https://drive.google.com/file/d/1G7hSVvKKvenRoq02h0ipsvTLef78aQwc/view?usp=sharing)
console: [link](https://drive.google.com/file/d/10ek1oJ3ZKPIWJfUjmZrdMQ-yHqN4KYN5/view?usp=sharing)

Am I doing something wrong?
All help is welcome
thanks

**Comment by jetannenbaum** [comment]
You are missing the Measure DHT11/22 Sensor block inside the loop.  See: https://bipes.net.br/wp/sample-page/

**Reply by triatebr** [reply]
Thanks for the tip, I made adjustments and now it's OK
blocks: [link](https://drive.google.com/file/d/159W7OxMKH_8So12yBC7U_9QSluXDq-h8/view?usp=sharing)

**Comment by jetannenbaum** [comment]
Happy to helpSent from Jet’s iPhoneOn Feb 16, 2023, at 7:56 AM, Lucas ***@***.***> wrote:﻿
Thanks for the tip, I made adjustments and now it's OK
blocks: link
